aiohttp/client_exceptions.py

```diff
@@ -175,13 +176,16 @@ class UnixClientConnectorError(ClientConnectorError):
     """
 
     def __init__(
-        self, path: str, connection_key: ConnectionKey, os_error: OSError
+        self,
+        path: Optional[Union[str, Path]],
+        connection_key: ConnectionKey,
+        os_error: OSError,
     ) -> None:
         self._path = path
         super().__init__(connection_key, os_error)
 
     @property
-    def path(self) -> str:
+    def path(self) -> Optional[Union[str, Path]]:
```

Comment:
I don't understand why you've just added `Optional` everywhere. You've now gone from 1 error to 32 errors.

Reply:
If you revert these changes, back to where you were before. There was only 1 error, it should be easy to then add the right annotation to pass the tests, you were almost there.

Reply:
i create a new pull request